qt/python/mantidqtinterfaces/test/Muon/PlottingView_test.py

```diff
@@ -1,420 +0,0 @@
-# Mantid Repository : https://github.com/mantidproject/mantid
-#
-# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
-#   NScD Oak Ridge National Laboratory, European Spallation Source,
-#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
-# SPDX - License - Identifier: GPL - 3.0 +
-import sys
-import unittest
-
-from matplotlib.figure import Figure
-from mantid import WorkspaceFactory, plots
-from unittest import mock
-from mantidqt.utils.qt.testing import start_qapplication
-
-from mantidqtinterfaces.Muon.GUI.ElementalAnalysis.Plotting.subPlot_object import subPlot
```

Comment:
https://github.com/mantidproject/mantid/tree/main/qt/python/mantidqtinterfaces/mantidqtinterfaces/MultiPlotting

Reply:
should it be in here https://github.com/mantidproject/mantid/tree/main/qt/python/mantidqtinterfaces/test/MultiPlotting